pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 	"reflect"
+	"strings"
```

Comment:
Please update the commit message and PR description to reflect all the changes you are doing here.

Reply:
Ack, I am going to stop re-concile always, as you suggested.